cmd/geth/chaincmd.go

```diff
@@ -91,6 +94,21 @@ Optional second and third arguments control the first and
 last block to write. In this mode, the file will be appended
 if already existing.`,
 	}
+	copydbCommand = cli.Command{
+		Action:    utils.MigrateFlags(copyDb),
+		Name:      "copydb",
+		Usage:     "Copy from one chain DB into another using the downloader",
+		ArgsUsage: "<filename>",
+		Flags: []cli.Flag{
+			utils.DataDirFlag,
+			utils.CacheFlag,
+			utils.SyncModeFlag,
+			utils.FakePoWFlag,
```

Comment:
I think we're going to need 

```
			utils.TestnetFlag,
			utils.RinkebyFlag,
```

too to allow copying over the testnet chains.